site/tracesdb-engine/index.html

```
@@ -0,0 +1,255 @@
+            <select id="qAttrOp" style="width:80px">
+              <option value="eq">=</option>
+              <option value="neq">≠</option>
+              <option value="gt">&gt;</option>
+              <option value="gte">≥</option>
+              <option value="lt">&lt;</option>
+              <option value="lte">≤</option>
+              <option value="contains">contains</option>
+              <option value="regex">regex</option>
+              <option value="exists">exists</option>
+            </select>
+            <input id="qAttrVal" type="text" placeholder="value" style="width:140px" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)